core/modules/util/Error.cc

```diff
@@ -0,0 +1,56 @@
+// -*- LSST-C++ -*-
+/*
+ * LSST Data Management System
+ * Copyright 2014 LSST Corporation.
```

Comment:
The copyright needs to be updated whenever a source file is modified.  It can't really be templated, as it needs to be in the original source as distributed via git.
(It's not entirely clear to me whether it should say "2008-2015" based on the entire DM system or "2014-2015" based on the particular file.  The "AURA/LSST" is important, though.)